changelogs/fragments/584-zos_lineinfile-error-message.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Fixed wrong error message when USS is not found, aligning with a similar error message from zos_blockinfile "{src} does not exist".
```

Comment:
I think it would be better to say "when a USS source is not found" instead of "when USS is not found".

Reply:
Agree!